### ISSUE TITLE
module 'aiohttp.web_urldispatcher' has no attribute 'mimetypes'

### ISSUE THREAD

**Issue by nicofd** [rckclmbr/pyportify#81]
Hi there!

I'm trying to run pyportify on Windows, using pyportify.exe or pyportify-copyall.exe (v0.3.6d).

I was able to copy some of my playlist using the _-copyall_ version. 
The process crashed on a playlist (unfortunately I didn't save the log), so I wanted to try the "not-copyall-version".

It starts without error, but when trying to access the webserver I got the following error :

```Listening on http://0.0.0.0:3132
Please open your browser window to http://localhost:3132
Error handling request
Traceback (most recent call last):
  File "site-packages\aiohttp\server.py", line 261, in start
  File "site-packages\aiohttp\web.py", line 88, in handle_request
  File "pyportify\dispatcher.py", line 68, in static_view
  File "asyncio\coroutines.py", line 206, in coro
  File "pyportify\dispatcher.py", line 43, in handle
AttributeError: module 'aiohttp.web_urldispatcher' has no attribute 'mimetypes'
Error handling request
```

Any idea about that ?

Thanks for the good work !

**Comment by CodyBurton** [comment]
Also getting this issue, Chrome version: 54.0.2840.99 m (64-bit)

**Comment by orthabel** [comment]
same here, also Chrome 54.0.2840.99 m (64-bit)

**Comment by rckclmbr** [comment]
This should be fixed in [v0.3.12](https://github.com/rckclmbr/pyportify/releases/v0.3.12/), let me know if that works for you.

**Comment by orthabel** [comment]
Works for me, thx!
